src/pybind/mgr/dashboard/tests/test_rest_client.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+import unittest
+
+from .. import mgr
+from ..rest_client import RestClient
+from mock import patch
+
+
+class RestClientTest(unittest.TestCase):
```

Comment:
Another idea would be to actually execute a network request, similar to @p-na 's proxy test. And test the exception message. But I'm OK with this one.

Reply:
Wanted to make it as simple as possible.